### ISSUE TITLE
Creating Time Slots (Easier)

### ISSUE THREAD

**Issue by DarkManX16** [vexorian/dizquetv#467]
Yes I noticed that creating weekly timeslots for everyday of the week is very tedious and time consuming.
I would like to request an easier/faster way to create weekly timeslots to cut down on the time it takes.
Thanks for your all hardwork on this program though ✊🏾

**Comment by Conan1996-2** [comment]
I found that when you do it for "Daily", set up all your time slots. Then, switch it to "Weekly" and it will copy ALL the time slots from "Daily" to every day under "Weekly". then you can modify each day as needed. Helpful if you wish to have the same show every day of the week, then you can modify for weekends, etc. Takes a few minutes but it is quicker then doing each day.

**Comment by DarkManX16** [comment]
You are absolutely right. I think I did this the first time I downloaded the program and I just completely forgot because I haven't done it in awhile
